app/controllers/issue_monitoring_reviews_controller.rb

```diff
@@ -0,0 +1,31 @@
+class IssueMonitoringReviewsController < ApplicationController
+  include NeedsHsesAccessToken
+
+  def create
+    @issue_monitoring_review = IssueMonitoringReview.new(
+      issue_id: issue_id,
+      review_id: review_id,
+      access_token: hses_access_token
+    )
+
+    respond_to do |format|
+      if @issue_monitoring_review.save
+        format.js { render inline: "location.reload(true);" }
+      else
+        Rails.logger.error("Problem saving the link to RAN review: " + @issue_monitoring_review.errors.full_messages.join("; "))
```

Comment:
[note] I can fill this sort of section out in this PR but I was hoping to save it for https://github.com/OHS-Hosting-Infrastructure/complaint-tracker/issues/207

Reply:
[suggestion] maybe a middle ground of `format.js { render inline: "alert('there was an error');" }`

Reply:
Good idea!